6-Typescript/server/controllers/todoController.ts

```diff
@@ -1,78 +1,66 @@
-import { ITodo } from './../interfaces/ITodo';
-import {Request, Response} from 'express'
-import { ITodoService } from '../interfaces/ITodoService';
+import { ITodo } from "../interfaces/interfaces";
+import { Request, Response } from "express";
+import { ITodoService } from "../interfaces/interfaces";
 export class TodoController {
-    db: ITodoService
+    private db: ITodoService;
     constructor(db: ITodoService) {
-        this.db = db
-
+        this.db = db;
     }
     getTodoById = async (req: Request, res: Response): Promise<void> => {
         const userId: string = req.body.userId;
-        const todoId: string= req.params.id
-        
-        const todo: ITodo | null = await this.db.modelGetTodoById(userId, todoId);
-        if(todo) {
-            res.status(200).json(todo)
-        } else {                                                
-            res.sendStatus(404)
+        const todoId: string = req.params.id;
+
+        const todo: ITodo | null = await this.db.modelGetTodoById(
+            userId,
+            todoId
+        );
+        if (todo == null) {
```

Comment:
why did you change the order?

Reply:
Tom told m to change from if else to just if. something like:
        if (todo == null) {
            res.sendStatus(404);
            return;
        }
        res.sendStatus(200);

        and not use else. So i changed it.

Reply:
sababa :)